mattermost.go

```diff
@@ -1370,6 +1376,48 @@ func cmdActivateUser() {
 	}
 }
 
+func cmdSlackImport() {
+	if flagCmdSlackImport {
+		if len(flagTeamName) == 0 {
+			fmt.Fprintln(os.Stderr, "flag needs an argument: -team_name")
+			flag.Usage()
+			os.Exit(1)
+		}
+
+		if len(flagImportArchive) == 0 {
+			fmt.Fprintln(os.Stderr, "flag needs an argument: -import_archive")
+			flag.Usage()
+			os.Exit(1)
+		}
+
+		var team *model.Team
+		if result := <-api.Srv.Store.Team().GetByName(flagTeamName); result.Err != nil {
+			l4g.Error("%v", result.Err)
+			flushLogAndExit(1)
+			_ = team
```

Comment:
Oops! That was when I wanted to run it when it was only half implemented, to stop go whinging about team never being used. Wasn't supposed to make it into the final cut.